docs/validators/delegation-manager.md

```diff
@@ -96,6 +102,49 @@ SetMetadataTransaction {
 }
 ```
 
+An example for the `Data` field that sets the name to `"Elrond Staking"`, the website to `"elrond.staking"` and the keybase.io identifier to `"elrondstaking"` is: 
+```
+    "setMetaData" + 
+    "@456c726f6e64205374616b696e67"     //Elrond Staking
+    "@656c726f6e642e7374616b696e67"     //elrond.staking     
+    "@656c726f6e647374616b696e67"       //elrondstaking
+```
+
+:::important
+Setting the keybase.io identity of the staking pool in the metadata is the **first step** in connecting the delegation contract and a keybase.io identity. The second step is explained in the next chapter [display information](/validators/delegation-manager#display-information) where the inverse connection is made: from the keybase.io identity to the delegation contract address. 
+:::
+
+### Display Information
+
+To customize the information for your delegation contract that will be available in the lists displayed on the delegation pages both in Maiar and the web wallet some aditional information has to be added on the keybase.io account. Please fill in the **avatar picture** and edit the profile providing the **name** and **bio**. This information together with the **service fee, percentage filled** and **APR** will be displayed for every delegation contract on the delegation pages in the web wallet and Maiar. If these information cannot be found a generic logo and the delegation contract's address is displayed.
+
+In order to complete the matching between the delegation contract and keybase.io identity of the staking pool an empty file with the name set to the delegation contract's address has to be added in the `/public/<keybase.io identity>/elrond/` folder. An example for the path to the empty file for the `"elrondstaking"` keybase.io identity would be `public/elrondstaking/elrond/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py`.
+
+:::important
+The **second step** in connecting the delegation contract and the keybase.io identity is finalized by adding the empty file that has the name equal to the delegation contract's address and certifying that the referenced keybase.io identity set in the [metadata](/validators/delegation-manager#metadata) also references the delegation contract address. This way the connection from both directions is sealed.
+:::
+
+:::tip
+To be able to connect a **testnet** or **devnet** contract to a keybase.io identity a new folder named `"testnet"` for the testnet or `"devnet"` for the devnet has to be created inside the `/elrond` folder. An example for the same delegation contract on the `testnet` would be `public/elrondstaking/elrond/testnet/erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqr0llllsj732py`.
```

Comment:
It might be better to put the path to the file as a code block using three backticks so it won't be wrapped when displayed by the browser.

Reply:
fixed